ci/terraform/account-management/shared.tf

```diff
@@ -19,4 +19,7 @@ locals {
   client_registry_encryption_key_arn          = data.terraform_remote_state.shared.outputs.client_registry_encryption_key_arn
   user_profile_kms_key_arn                    = data.terraform_remote_state.shared.outputs.user_profile_kms_key_arn
   email_check_results_encryption_policy_arn   = data.terraform_remote_state.shared.outputs.email_check_results_encryption_policy_arn
+
+  slack_event_sns_topic_arn = data.terraform_remote_state.shared.outputs.slack_event_sns_topic_arn
```

Comment:
As previous, potentially the same missing `dynatrace_secret` here too.

Reply:
https://github.com/govuk-one-login/authentication-api/pull/5644#discussion_r1903942594